src/kvirc/sparser/KviIrcServerParser_literalHandlers.cpp

```diff
@@ -1787,11 +1787,13 @@ void KviIrcServerParser::parseLiteralNick(KviIrcMessage *msg)
 			old->mergeQuery(q);
 			g_pMainWindow->closeWindow(q); // deleted path
 			if(!msg->haltOutput())
-					old->output(KVI_OUT_NICK,__tr2qs("\r!n\r%Q\r [%Q@\r!h\r%Q\r] is now known as \r!n\r%Q\r"),
+				old->output(KVI_OUT_NICK,__tr2qs("\r!n\r%Q\r [%Q@\r!h\r%Q\r] is now known as \r!n\r%Q\r"),
```

Comment:
@DarthGandalf this is also right, if its not then look at the rest of the file and if this is wrong then the rest just like these are also wrong

there was one to many tabs here